gaia/parser.py

```diff
@@ -35,12 +40,29 @@
     gaia.geo.geo_inputs, inspect.isclass) if x[0].endswith('IO')])
 
 
+def load_plugins():
```

Comment:
> @aashish24 I can move the load_plugins() definition to gaia/init.py or gaia/core.py, do you have a preference?

core.py sounds better to me. 



> twitter plugin
> stats plugin (anything that depends on the pysal module)
> maybe a raster plugin too, but keep the vector io's/processes in geo as core functionality?

+1, that's what I was thinking. May be we can keep the base classes for raster and vector in core and move the actual implementations as plugins?